### PR TITLE
INT B-20651 Prime Counseling Completed in Move History

### PR DESCRIPTION
**Pull request by ajlusk**
## Summary

Updates the move history to display details about prime counseling completion. The two items displayed below both used to use the generic template and only showed an em dash in the description.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

#### Set up a prime counseled move
1. Login as a new customer.
2. Create a new move, setting your origin duty location to NS Mayport (ZIP:32228) or some other duty location that does not offer services counseling.
3. Add a PPM shipment (HHG will only use one of the two added templates).
4. Optionally also add an HHG.
5. Submit your move.
6. Log in as a TOO (NS Mayport is in CNNQ but for me the move didn't show up in any GBLOC's TOO queue, looking into if that needs to be documented as an issue).
7. Open the move you created via move search.
8. Enter in required fields and approve the move (The summary lists no service items but maybe it should list prime counseling, looking into that as well).

#### Prime counseling
1. Log into the prime simulator and open the move created and approved above.
2. Click "Complete counseling"
3. Verify a counseling completed timestamp displays
4. Log back into the office app as any user and open the move.
5. Verify the move history contains two entries like the screenshot below. 

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

## Screenshots
Screenshot 2024-07-29 at 1 03 34 PM

### CODE REVIEW

File: src/constants/MoveHistory/EventTemplates/UpdateMTOPostCounselingInformation/updateMTOStatusPrimeCounselingCompletedPPM.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+};
+
+export default {
+  action: a.UPDATE,
+  eventName: o.updateMTOPostCounselingInformation,
+  tableName: t.ppm_shipments,
+  getEventNameDisplay: () => 'Updated shipment',
+  getDetails: (historyRecord) => <LabeledDetails historyRecord={formatChangedValues(historyRecord)} />,
+};
+
```
Comment (MInthavongsay) [inline src/constants/MoveHistory/EventTemplates/UpdateMTOPostCounselingInformation/updateMTOStatusPrimeCounselingCompletedPPM.jsx:25]: Delete commented out code before merging.

**Review by MInthavongsay — APPROVED** [review]
Looks good. Verified two history entries were displayed per instructions.

File: src/constants/MoveHistory/EventTemplates/UpdateMTOPostCounselingInformation/updateMTOStatusPrimeCounselingCompletedPPM.jsx (hunk 1 above)
Comment (ajlusk) [reply src/constants/MoveHistory/EventTemplates/UpdateMTOPostCounselingInformation/updateMTOStatusPrimeCounselingCompletedPPM.jsx:25]: Good catch, thanks!

**Review by antgmann — APPROVED** [review]
Looks great! Only possible nitpick I could possibly come up with is multiple shipments might seem to mess with the move history order of events, but I don't think this is an issue especially due to the presence of the timestamp. Nicely done!
image
image

**Review by CoryKleinjanCACI — APPROVED** [review]

**Comment by ajlusk** [comment]
> Looks great! Only nitpick I could possibly come up with is multiple shipments might seem to mess with the move history order of events, but I don't think this is an issue especially due to the presence of the timestamp. Nicely done! image image

Fair concern. Looks like the endpoint updates the move and all shipments in the same transaction. Doesn't seem within the scope of this ticket to fiddle with that to ensure a specific order.
Screenshot 2024-08-01 at 10 16 41 AM

**Pull request merged**